### PR TITLE
Change recipe so that the osx can use the sasl c-extension

### PR DESCRIPTION
**Pull request by mariusvniekerk**

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,19 +13,20 @@ source:
...
     - setuptools
+    - toolchain
 
   run:
     - python
     - thrift >=0.10
-    - pure-sasl  # [not linux]
-    - sasl  # [linux] 
+    - pure-sasl  # [win]
+    - sasl  # [osx]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Should it be `unix`?

Comment (mariusvniekerk) [reply recipe/meta.yaml]: yep

**Comment by mariusvniekerk** [comment]
@conda-forge-admin, please rerender

**Pull request merged**